packages/hash/api/codegen.yml

```diff
@@ -23,6 +23,9 @@ generates:
       noSchemaStitching: true
       skipTypename: true
       scalars:
+        DataType: object
+        PropertyType: object
+        LinkType: object
```

Comment:
Not sure if I've fully understood, the PR description suggests we can't type these because the `@hashintel/hash-graph-client` hasn't been added to the frontend. Why not just add `@hashintel/hash-graph-client` to the frontend then? Even if all we use is the types?

Reply:
There isn't anything stopping us from doing that I suppose. It was more that I didn't want to leak the SDK to the frontend. If we're okay with doing that, I can add it

Reply:
I don't think we should sacrifice type safety just to stop leaking the SDK to the frontend. Ideally we would have the types split into a dedicated package, but I don't think we should worry about that now as it seems like how these types are generated will change anyway if we move away from REST.

Reply:
Let's add a TODO to undo this when we have the types in a more accessible shared location

Reply:
Added in d3450ab43e4b02b95165244aa82fdd8cee8d2cff